fe/fe-core/src/main/java/com/starrocks/external/iceberg/cost/IcebergColumnStats.java

```diff
@@ -0,0 +1,72 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.external.iceberg.cost;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class IcebergColumnStats {
```

Comment:
What's the difference `IcebergColumnStats` from `HiveColumnStats`

Reply:
HiveColumnStats has strong correlation with hms, IcebergColumnStats only contains min/max/avb/numnulls and do not support update logic now